src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/ProjectInitializationHandler.cs

```diff
@@ -32,7 +31,7 @@ internal class ProjectInitializationHandler : IDisposable
 
     [ImportingConstructor]
     [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-    public ProjectInitializationHandler([Import(typeof(SVsFullAccessServiceBroker))] IServiceBroker serviceBroker, ILoggerFactory loggerFactory)
+    public ProjectInitializationHandler([Import("Microsoft.VisualStudio.Shell.ServiceBroker.SVsFullAccessServiceBroker")] IServiceBroker serviceBroker, ILoggerFactory loggerFactory)
```

Comment:
Can we introduce a constant or something similar to avoid the magic string here?

TBH I'd rather get rid of the symbol banning entirely then change to this, since I'm not sure what value is being provided.

Reply:
> Can we introduce a constant or something similar to avoid the magic string here?

Yes, a constant would be possible.

> TBH I'd rather get rid of the symbol banning entirely then change to this, since I'm not sure what value is being provided.

There is no representation of this import because it was exported with an explicit name instead of its type (and the name it was exported with does not intersect with its implemented types).

Reply:
➡️ Updated to use a constant